pythnet/remote-executor/programs/remote-executor/src/state/posted_vaa.rs

```diff
@@ -0,0 +1,43 @@
+use std::{io::Write, str::FromStr, ops::Deref};
+use anchor_lang::prelude::*;
+use wormhole_solana::VAA;
+
+impl Owner for AnchorVaa {
+    fn owner() -> Pubkey{
+        Pubkey::from_str("H3fxXJ86ADW2PNuDDmZJg6mzTtPxkYCpNuQUTgmJ7AjU").unwrap() // Pythnet bridge address
```

Comment:
Could we avoid hardcoding this?

Reply:
yeah this seems like it should be configurable at deployment time or something.

Reply:
https://github.com/pyth-network/pyth-crosschain/pull/289